src/tools/cephfs/cephfs-shell

```diff
@@ -773,6 +773,129 @@ class CephFSShell(Cmd):
+                if is_file_exists(args.source_path):
+                    dest = args.destination_path.decode('utf-8').rstrip('/')
+                    dest_first_half = dest.encode('utf-8') + b'/'
+                    file = re.findall(r"[^/]+(?=/$|$)",
+                                      args.source_path.decode('utf-8'))
+                    if len(file) == 1:
+                        dest_second_half = file[0].encode('utf-8')
+                        destination_path = dest_first_half + dest_second_half
+                        # print("file -> dir dest path: ", destination_path)
+
+                elif is_dir_exists(args.source_path):
+                    dir1 = re.findall(r"[^/]+(?=/$|$)",
+                                      args.source_path.decode('utf-8'))
+                    dir2 = re.findall(r"[^/]+(?=/$|$)",
+                                      args.destination_path.decode('utf-8'))
+                    if dir1[0] == dir2[0]:
+                        destination_path = args.destination_path
+                    else:
+                        dest = args.destination_path.decode('utf-8').rstrip('/')
+                        dest_first_half = dest.encode('utf-8') + b'/'
+                        dest_second_half = dir1[0].encode('utf-8')
+                        destination_path = dest_first_half + dest_second_half
+                    # print("dir -> dir dest path: ", destination_path)
+
+            # elif is_file_exists(args.source_path) and \
+            #         (is_dir_exists(args.destination_path) or is_file_exists(args.destination_path)):
+            #     destination_path = args.destination_path
+            #     # print('dest path', destination_path)
+
+            else:
+                test_path = args.destination_path.decode('utf-8')
+                if test_path[len(test_path)-1] == '/':
+                    set_exit_code_msg(errno.ENOENT, f"'{test_path}': "
+                                                    f"No such file or directory")
+                    return
+                else:
+                    destination_path = test_path
+                # print('final dest: ', destination_path)
+
+        if args.symbolic:
+            if args.force:
+                try:
+                    cephfs.unlink(destination_path)
+                except libcephfs.Error:
+                    pass
+
+            cephfs.symlink(source_path, destination_path)
+
+            if args.verbose:
+                poutput(f"{destination_path.decode('utf-8')} ->"
+                        f" {source_path.decode('utf-8')}")
+        else:
+            if args.force:
+                try:
+                    cephfs.unlink(destination_path)
+                except libcephfs.Error:
+                    pass
+
+            cephfs.link(source_path, destination_path)
+
+            if args.verbose:
+                poutput(f"{destination_path.decode('utf-8')} ->"
+                        f" {source_path.decode('utf-8')}")
+
```

Comment:
```suggestion
        if args.force:
                try:
                    cephfs.unlink(destination_path)
                     if args.symbolic:
                        cephfs.symlink(source_path, destination_path)
                     else:
                        cephfs.link(source_path, destination_path)
                except libcephfs.Error:
                    # print errmsg and errno here?
                    return

        if args.verbose:
            poutput(f"{destination_path.decode('utf-8')} ->"
                    f" {source_path.decode('utf-8')}")
```

Reply:
Okay

Reply:
Done

Reply:
No. This one hasn't been covered yet. You are still using 2 try-excepts and `if args.force` which is only duplication.

Reply:
Perhaps you forgot to add and push this change?

Reply:
Ah my bad, i was doing some changes and forgot to push it. I'll do it